datumaro/plugins/vott_csv_format.py

```diff
@@ -0,0 +1,78 @@
+# Copyright (C) 2021 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+
+import csv
+import os.path as osp
+
+from datumaro.components.annotation import AnnotationType, Bbox, LabelCategories
+from datumaro.components.extractor import DatasetItem, Importer, SourceExtractor
+from datumaro.components.format_detection import FormatDetectionContext
+from datumaro.util.image import find_images
+from datumaro.util.meta_file_util import has_meta_file, parse_meta_file
+
+
+class VottCsvPath:
+    ANNO_FILE_SUFFIX = '-export.csv'
+
+class VottCsvExtractor(SourceExtractor):
+    def __init__(self, path):
+        if not osp.isfile(path):
+            raise FileNotFoundError("Can't read annotation file '%s'" % path)
+
+        super().__init__(subset=osp.splitext(osp.basename(path))[0].split('-')[0])
```

Comment:
The split will not behave correctly if the subset name contains dashes. You should use `rsplit` with `maxsplit=1` instead.